app/src/main-process/menu/build-default-menu.ts

```diff
@@ -165,7 +165,7 @@ export function buildDefaultMenu(
       {
         label: '&Reload',
         id: 'reload-window',
-        accelerator: 'CmdOrCtrl+Alt+R',
+        accelerator: 'CmdOrCtrl+Shift+R',
```

Comment:
Since this one's just available on dev builds, I'd suggest keeping it `Alt`. That way we can use `CmdOrCtrl+Shift+R` for a user-facing feature down the road if needed.